.gitmodules

```diff
@@ -6,3 +6,7 @@
 	path = submodules/Common
 	url = https://github.com/aspnet/Common.git
 	branch = NuGet/SubModuleIntegration
+[submodule "submodules/NuGet.Services.Metadata"]
+	path = submodules/NuGet.Services.Metadata
+	url = https://github.com/NuGet/NuGet.Services.Metadata.git
+	branch = searchcloud
```

Comment:
Is this branch just for the client? If not you should make a special branch on NuGet.Services.Metadata to avoid any issues. With the other submodules we had to do this to avoid breaking changes, the commit saved didn't seem to stop it.